AWS/centos/centos.json

```diff
@@ -88,13 +88,15 @@
       "local_state_tree": ".tmp/{{ user `distro_slug` }}/sre-jenkins/states",
       "local_pillar_roots": ".tmp/{{ user `distro_slug` }}/sre-jenkins/pillar",
       "custom_state": "services.jenkins-node.ec2-latent-slave",
-      "bootstrap_args": "{{ user `bootstrap_args` }} git v{{ user `bootstrap_version` }}"
+      "bootstrap_args": "{{ user `bootstrap_args` }} stable {{ user `bootstrap_version` }}"
```

Comment:
Latest stable, when I started creating golden images was problematic, let's hope it ain't anymore

Reply:
it's 2018.3.4 stable